Tools/Create-Azure-Sentinel-Solution/V2/WorkbookMetadata/WorkbooksMetadata.json

```diff
@@ -5511,5 +5511,20 @@
     "templateRelativePath": "MailGuard365Dashboard.json",
     "subtitle": "",
     "provider": "MailGuard 365"
+},
+{
+    "workbookKey": "MimecastTIRegionalWorkbook",
+    "logoFileName": "Mimecast.svg",
+    "description": "A workbook providing insights into Mimecast Regional Threat indicator.",
+    "dataTypesDependencies": ["ThreatIntelligenceIndicator"],
```

Comment:
Hello @nipun-crestdatasystem, Please add dataconnector dependency property in the workbook metadata. You refer the workbook metadata declared above.